generators/module/templates/eslintrc.yml

```diff
@@ -1,3 +1,6 @@
 extends: cheminfo
 parserOptions:
   sourceType: module
+env:
+  node: true
+  jest: true
```

Comment:
It is deliberate to not put these:
- `node: true` adds globals and stuff that don't work in browsers
- `jest: true` is not relevant in all projects

Reply:
Ok for node indeed we should not add it

But the generators add `jest` by default so currently it fails to pass the tests because we don't have this `env: jest: true`. Is it really not relevant for all projects created by the generator ?

Reply:
Ok if Jest is added in the project, we should have it here.